cache-config/t3c-request/config/config.go

```diff
@@ -83,7 +82,7 @@ func InitConfig() (Cfg, error) {
 		Usage()
 	}
 	if *versionPtr == true {
-		fmt.Println(AppName + " v" + Version)
+		log.Errorln(AppName + " v" + Version)
```

Comment:
Like with the other app, the loggers haven't been initialized here yet. You can either call `log.Init(os.Stderr, os.Stderr, os.Stderr, os.Stderr, os.Stderr)`, or just print since it's highly unlikely`--version` calls will ever need run thru a log parser.

Reply:
Put back to print.